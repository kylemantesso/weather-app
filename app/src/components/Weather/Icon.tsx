import React from "react";

export const Icon = ({
  icon,
  description,
}: {
  icon: string;
  description: string;
}) => (
  <img
      alt={description}
    src={`https://www.metaweather.com/static/img/weather/${icon}.svg`}
    height="60px"
  />
);
