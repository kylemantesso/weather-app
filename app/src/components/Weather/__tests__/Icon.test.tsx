import React from "react";
import renderer from "react-test-renderer";
import {Icon} from "../Icon";

describe("Icon", () => {
  it("matches snapshot", () => {
    const component = renderer.create(
      <Icon
       icon="icon"
       description="description"/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
