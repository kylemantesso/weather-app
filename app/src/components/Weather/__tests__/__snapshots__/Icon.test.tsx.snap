// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon matches snapshot 1`] = `
<img
  alt="description"
  height="60px"
  src="https://www.metaweather.com/static/img/weather/icon.svg"
/>
`;
